src/elements/content-sidebar/activity-feed/task-new/__tests__/__snapshots__/TaskStatus.test.js.snap

```diff
@@ -5,18 +5,20 @@ exports[`elements/content-sidebar/ActivityFeed/task-new/TaskStatus should render
   status="APPROVED"
 >
   <FormattedMessage
-    defaultMessage="Status: {taskStatus}"
+    defaultMessage="{taskStatus}"
```

Comment:
Double-check the values here use the upper-case variant.

Reply:
removing this message entirely as the taskStatus can be rendered directly